src/pages/learn.js

```diff
@@ -359,8 +364,44 @@ const LearnPage = ({
         }
     };
 
-    const { page } = filterValue;
+    /**
+     * Renders a CardList component with the filter results
+     * @returns A CardList Component
+     */
+    const ActiveCardListFilter = () => {
+        switch (activeContentTab) {
+            case LearnPageTabs.articles:
+                return (
+                    <CardList
+                        articles={textFilterResults}
+                        textFilterQuery={textFilterQuery}
+                    />
+                );
+            case LearnPageTabs.videos:
+                return (
+                    <CardList
+                        videos={textFilterResults}
+                        textFilterQuery={textFilterQuery}
+                    />
+                );
+            case LearnPageTabs.podcasts:
+                return (
+                    <CardList
+                        podcasts={textFilterResults}
+                        textFilterQuery={textFilterQuery}
+                    />
+                );
+            default:
+                return (
+                    <CardList
+                        all={textFilterResults}
+                        textFilterQuery={textFilterQuery}
+                    />
+                );
+        }
```

Comment:
Nice! . can we abstract this into another file.

Reply:
except for PodcastId everything else looks good to me.Great work as always.

Reply:
Would it make sense to use _id instead of podcastId as the field name since that is what we have for articles as well?

Reply:
> Nice! . can we abstract this into another file.

For some reason, when I tried to abstract to another file, the app re-renders this twitch video `6 Things Lauren Learned While Modeling Data in MongoDB` on each tab as the first card. I've been on this for a while and cannot seem to figure out why it does that

Reply:
let me take a look at this.